tracer/tracer.go

```diff
@@ -749,6 +641,18 @@ func loadProgram(ebpfProgs map[string]*cebpf.Program, tailcallMap *cebpf.Map,
 	return nil
 }
 
+func (t *Tracer) pollKobjectClient() {
+	for {
+		event, err := t.kobjectClient.Receive()
+		if err != nil {
+			return
```

Comment:
Should we log this error? There's nothing logged in `updateSymbolsFrom` otherwise